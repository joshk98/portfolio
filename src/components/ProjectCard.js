import React from "react";

import "../styles/projectcard.css";

function ProjectCard() {
  return <div>ProjectCard</div>;
}

export default ProjectCard;
