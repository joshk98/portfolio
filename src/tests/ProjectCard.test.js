import React from "react";
import { render } from "@testing-library/react";
import ProjectCard from "../components/ProjectCard";

describe("ProjectCard", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<ProjectCard />);

    expect(asFragment()).toMatchSnapshot();
  });
});
