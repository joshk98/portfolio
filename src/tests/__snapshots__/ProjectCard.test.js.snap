// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectCard renders correctly 1`] = `
<DocumentFragment>
  <div>
    ProjectCard
  </div>
</DocumentFragment>
`;
